src/features/game/types/craftables.ts

```diff
@@ -445,7 +456,37 @@ export const BLACKSMITH_ITEMS: Record<BlacksmithItem, Craftable> = {
       },
     ],
     supply: 100000,
-    disabled: true,
+  },
+
+  "Bee Hive": {
+    name: "Bee Hive",
+    description: "Gather 20% more honey",
```

Comment:
The PR description says the hive unlocks honey production (meaning > 0).  The description for the item says "20% more".  Which is it?

Reply:
Just noticed it was a mistake that's supposed to be the Bee Box ( that is still not available anywhere to craft yet )